### PR TITLE
creating scripts to take manual observations - WIP

### PR DESCRIPTION
**Pull request by lspitler**

### CODE REVIEW

File: scripts/take_day_flats.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        print(f"Starting exposures {i:03d} of {num_exposures:03d}.")
+
+        exposure_events = []
+
+        # Share timestamp filename across all cameras.
+        timestamp = current_time(flatten=True)
+        for cam_name, camera in cameras.items():
+            new_filename = f'{base_dir}/{base_date}/{camera.uid}/{timestamp}.fits'
+
+            exposure_events.append(camera.take_observation(observation, filename=new_filename))
```
Comment (JAAlvarado-Montes) [inline scripts/take_day_flats.py]: Need to add `headers={}` inside `camera.take_observation` to avoid a bug in POCS.

**Review by JAAlvarado-Montes — CHANGES_REQUESTED** [review]
Look at the comment above, include `headers={}` to avoid POC's bug

File: scripts/take_day_flats.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    for i in range(num_exposures):
+        print(f"Starting exposures {i:03d} of {num_exposures:03d}.")
+
+        exposure_events = []
+
+        # Share timestamp filename across all cameras.
+        timestamp = current_time(flatten=True)
+        for cam_name, camera in cameras.items():
+            new_filename = f'{base_dir}/{base_date}/{camera.uid}/{timestamp}.fits'
+
```
Comment (JAAlvarado-Montes) [inline scripts/take_day_flats.py:26]: Before doing `camera.take_observation`, it'll be good to move the filterwheel to the filter we want to use by using `camera.filterwheel.move_to(f'{filter_name}')`, so we need to add `filter_name` as an argument to the function `take_pics`. Currently, we have to manually move the filterwheel of each camera, thawhich could be done inside this function automatically for all the cameras.

**Review by JAAlvarado-Montes — CHANGES_REQUESTED** [review]
See comments above.

**Comment by JAAlvarado-Montes** [comment]
I have made two changes to this PR, namely added `headers={}` to avoid a current bug in POCS and added a new function called `move_all_cameras_to_filter` to move all of the filterwheels to the same slot we want to use afterwards when using `take_pics`. Please review.

Cheers!

**Comment by lspitler** [comment]
Thanks @JAAlvarado-Montes for the edits. Do we need to add a function that turns on cooling for the cameras?

**Comment by wtgee** [comment]
FYI, this is on hold for just a bit but all these parts will probably be incorporated.

**Comment by AnthonyHorton** [comment]
Hopefully relevant to this:

https://github.com/panoptes/POCS/pull/1003 & #208

**Comment by AnthonyHorton** [comment]
> I have made two changes to this PR, namely added `headers={}` to avoid a current bug in POCS...

That's a reference to this: https://github.com/panoptes/POCS/issues/1002

**Comment by lspitler** [comment]
closing as this is being done in new repo `huntsman-bridge`

**Pull request closed without merging**